suites/PerformanceTestSuite.js

```
@@ -182,6 +257,21 @@ class PerformanceTestSuite extends TestSuite {
                 
                 if (fs.existsSync(swPath)) {
                     const content = fs.readFileSync(swPath, 'utf8');
+                    const analyzer = new PerformanceAnalyzer();
+                    
+                    // Service worker specific analysis
+                    analyzer.detectMemoryLeaks(content, path.basename(swPath));
+                    analyzer.detectHeavyComputation(content, path.basename(swPath));
+                    
+                    // Get any issues found
+                    const swIssues = analyzer.issues.filter(i => i.file === path.basename(swPath));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)